modules/plugins/postgres/src/main/java/org/rhq/plugins/postgres/PostgresDatabaseComponent.java

```diff
@@ -342,4 +346,80 @@ private OperationResult invokeSql(Configuration parameters) throws SQLException
         }
     }
 
+    /**
+     * This functionality is a port of vacuumlo tool in the Postgres distribution
+     *
+     * @return OperationResult
+     */
+    private OperationResult vacuumLo() {
+        Connection c = null;
+        Statement clearVacuumL = null;
+        Statement unlinkStatement = null;
+        ResultSet rs = null;
+        try {
+            c = getPooledConnectionProvider().getPooledConnection();
+            PreparedStatement tempTable = c.prepareStatement("CREATE TEMP TABLE vacuum_l AS (SELECT DISTINCT loid AS lo FROM pg_largeobject)");
```

Comment:
All standard use-cases prevent access to pg_largeobject by a non-superuser. This is true as of PostgreSQL 9.0 and later.

`ERROR: permission denied for relation pg_largeobject`

I am assuming in your testing you have given admin/superuser rights to your rhqadmin database user or have disabled permission checking altogether or have given explicit grants to LARGE OBJECT?